src/store/modules/layers.store.js

```diff
@@ -149,6 +149,22 @@ const getters = {
     getActiveLayersById: (state) => (layerId) =>
         state.activeLayers.filter((layer) => layer.id === layerId),
 
+    /**
+     * Retrieves layer(s) by ID.
+     *
+     * Search in active layer and in preview layer
+     *
+     * @param {string} layerId ID of the layer(s) to retrieve
+     * @returns {[AbstractLayer]} All active layers matching the ID
+     */
+    getLayersById: (state) => (layerId) => {
```

Comment:
might be good to rename that to `getActiveLayerById`, as it stands it could make you think it will lookup the whole layers config, but in reality will only seek matches in the active layers (preview layer is kind of an active layer too...)

Reply:
@pakb that is not a good idea, the catalogue uses `getActiveLayersById` to add/remove layer from the active layer list, here we definitely don't want to have the `previewLayer`